mono/metadata/class-internals.h

```diff
@@ -755,17 +755,17 @@ typedef struct {
 
 typedef struct {
 	guint64 new_object_count;
-	size_t initialized_class_count;
-	size_t generic_vtable_count;
+	gsize initialized_class_count;
```

Comment:
Is there any reason to prefer glib types to standard C types in internal headers?